slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -128,9 +144,8 @@ void SLAMServiceImpl::CreateMap() {
 
```

Comment:
Is the result of `listFileInDirectory()` sorted?

Reply:
yes, it's sorted :)